scripts/Index.md

```diff
@@ -0,0 +1,32 @@
+#  List of available Scripts
```

Comment:
Please name the file only with lower case letter. Add a short introduction, such as below the list of available scripts that can be started via the REST API. To learn more about the script and process endpoints are available here
* script link
* process link